samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -307,6 +312,9 @@ DEFINE_string(api, "async", api_message);
 /// @brief Number of infer requests in parallel
 DEFINE_uint64(nireq, 0, infer_requests_count_message);
 
+/// @brief Execute infer requests at a fixed frequency
+DEFINE_double(rfreq, 0, run_frequency_message);
```

Comment:
The default could be infinity. That way you don't need to handle 0 separately.

Reply:
If set to INFINITY, then the function `if (FLAGS_rfreq > 0)` will still get executed which is redundant right? Having default as 0 would simply skip that and runs without delay. What do you think?

Reply:
I don't know for sure. You should try. I'd expect sleep_for to have the similar logic incide: if <=0 then return.

Reply:
in C++ sleep_for does have the function to return if value is <= 0. But the `time.sleep()` in python cannot take negative value, so a logic to ensure <=0 is needed. To align both C++ and python style, I will keep this logic remained for now. let me know if there's other thought.